### DISCUSSION TITLE
# WARNING this tool not being optimal for very high memory systems

### DISCUSSION THREAD

**Discussion by sonnguyen-dba** [le0pard/pgtune#57] [General]
Hi Alexey Vasiliev,
Can you support "optimal for very high memory systems"?
memory > 3000G
Regards,
Son

**Comment by le0pard** [comment]
@sonnguyen-dba how much a memory mean "> 3000 G"? Like 5000 GB RAM?

For workstations, the biggest amount of RAM that you can take is around 2TB (For AMDs TR4, X399 chipsets). What kind hardware from 2021 support more, than 3000 GB RAM?

P.S. I ready to support high capacity systems, just provide me math model with test for all combinations of params. Thanks.

**Comment by sonnguyen-dba** [comment]
Hi @le0pard, I using Oracle X8-8 3TB RAM. 
P.S. I ready to support high capacity systems => yes.

Thanks

**Reply by le0pard** [reply]
Ok, but what mean "> 3000 G"? How much memory utility need to support?

**Reply by sonnguyen-dba** [reply]
thanks, It mean from ("> 3000 G") 3TB.

**Comment by le0pard** [comment]
You can send "math model with test for all combinations of params" in this thread.

**Reply by sonnguyen-dba** [reply]
I see warning on your web
# WARNING
# this tool not being optimal
# for very high memory syste
<img width="1136" alt="Screen Shot 2021-11-01 at 21 21 50" src="https://user-images.githubusercontent.com/72149376/139686975-b409c5fc-413d-4b6c-88ba-37b5bdab5a7c.png">
ms


Thanks, I think we shouldn't waste any more time on this.
Thanks you very much.

**Reply by le0pard** [reply]
Yep, warning is correct. I don't have tests and numbers for high RAM systems for PostgreSQL. So that is why I need math model for this

**Reply by sonnguyen-dba** [reply]
Thanks @le0pard, I see! Best wishes to you!
